crates/build/src/util/mod.rs

```diff
@@ -22,37 +22,12 @@ use anyhow::{
     Context,
     Result,
 };
-use rustc_version::Channel;
-use semver::Version;
 use std::{
     ffi::OsStr,
     path::Path,
     process::Command,
 };
 
-/// This makes sure we are building with a minimum `stable` toolchain version.
-fn assert_channel() -> Result<()> {
-    let meta = rustc_version::version_meta()?;
-    let min_version = Version::new(1, 63, 0);
-    match meta.channel {
-        Channel::Stable if meta.semver >= min_version => Ok(()),
```

Comment:
This min version check can be replaced by adding [`rust_version`](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field) to the `ink` crate, might be a more suitable location for it?